src/Controller/RegisterFrontendUsersController.php

```diff
@@ -73,7 +73,7 @@ public function save(?User $user, ValidatorInterface $validator): Response
             return $this->redirect($referer);
         }
 
-        $user = UserRepository::factory('', $this->request->get('username'), $this->request->get('email'));
+        $user = UserRepository::factory($this->request->get('displayname'), $this->request->get('username'), $this->request->get('email'));
```

Comment:
Can we make the `displayname` optional here?
For example:
`$this->request->get('displayname', '')`?

Reply:
@I-Valchev , I've given this a go, and it appears that setting it up as you suggested interrupts the registration flow, and the user isn't registered.

Reply:
Ah, my bad there! Thanks!